dateutil/rrule.py

```diff
@@ -347,11 +347,11 @@ class rrule(rrulebase):
         YEARLY, an interval of 2 means once every two years, but with HOURLY,
         it means once every two hours. The default interval is 1.
     :param wkst:
-        The week start day. Must be one of the MO, TU, WE constants, or an
-        integer, specifying the first day of the week. This will affect
-        recurrences based on weekly periods. The default week start is got
-        from calendar.firstweekday(), and may be modified by
-        calendar.setfirstweekday().
+        The week start day. Must be one of the constants (MO, TU, WE, etc) or 
```

Comment:
I think if MO, TU, WE, etc are going to be in parentheses, the rest of the sentence still has to work if the text were missing, so:

"Must be one of the constants or an integer"

I think maybe change to "Must be one of the weekday constants (MO, TU, WE, etc)"